std/bundle/README.md

```diff
@@ -1,15 +1,7 @@
 # bundle
 
-These are modules that help support bundling with Deno.
-
-## Usage
-
-The main usage is to load and run bundles. For example, to run a bundle named
-`bundle.js` in your current working directory:
-
-```sh
-deno run https://deno.land/std/bundle/run.ts bundle.js
-```
+As of Deno 0.24, bundles are generated with the loader/runner integrated into
+the bundle, therefore there are no `std` modules required.
```

Comment:
I'm just going to delete the directory. We're in unstable mode still, so I don't it's necessary to leave this.